Dockerfile

```diff
@@ -37,6 +37,7 @@ RUN \
 
 COPY package*.json /code/
 
+RUN npm install webpack -g
```

Comment:
Just noting here that my system seems to hang on this step. We may need to note in the install documentation that if the installation seems to hang up here, users should do an npm install before docker-compose up.